main.go

```diff
@@ -17,7 +17,7 @@ func main() {
 	flag.Parse()
 	err := providerserver.Serve(context.Background(), commonfate.New, providerserver.ServeOpts{
 		Debug:   debug,
-		Address: "commonfate.com/commonfate/commonfate",
+		Address: "registry.terraform.io/common-fate/commonfate",
```

Comment:
Im surprised we didn't need to change this earlier. Were you running into issues running it locally with the dev override?